tests/sv/cover_HassCloseCover.yaml

```diff
@@ -0,0 +1,37 @@
+language: sv
+tests:
+  - sentences:
+      - "stäng cloth"
+    intent:
+      name: "HassCloseCover"
+      slots:
+        name: cover.cloth
+  - sentences:
+      - "stäng cloth i Vardagsrum"
+    intent:
+      name: "HassCloseCover"
+      slots:
+        name: cover.cloth
+        area: living_room
+
+  # Garage door
+  - sentences:
+      - "stäng garagedörren"
+    intent:
+      name: "HassCloseCover"
+      slots:
+        device_class: "garage_door"
+
+  # Blinds
+  - sentences:
+      - "stäng gardin i Kök"
```

Comment:
This would probably be better as "Stäng gardinen i köket"

Reply:
We're trying to collect as many sentences for an intent as possible. One being better means we should include both.

Reply:
Ok, sure. Just so you understand the difference here; "Stäng gardin i Kök" translates to "Close curtains in kitchen". "The" is missing. So I guess you are right, we should include it anyway.